src/sentry/conf/server.py

```diff
@@ -1990,6 +1990,14 @@ def custom_parameter_sort(parameter: dict) -> tuple[str, int]:
 # that is earlier in the chain cannot manage the settings of a member later
 # in the chain (they still require the appropriate scope).
 SENTRY_ROLES: tuple[RoleDict, ...] = (
+    {
+        "id": "billing",
+        "name": "Billing",
+        "desc": "Can manage subscription and billing details.",
+        "scopes": {"org:billing"},
+        "is_team_roles_allowed": False,
+        "is_retired": False,
+    },
```

Comment:
Adding this role will expose this for the self-hosted open source instances of Sentry.

Reply:
Removed the `billing` role in [dc79aba](https://github.com/getsentry/sentry/pull/80129/commits/dc79aba6a2de89b21b83e7437b52f33b54babacb). The role was being used for validation purposes. The refactored validation logic will inspect the feature flags and determine if a bypass is needed.